doc/_templates/delivery.rst

```diff
@@ -0,0 +1,158 @@
+of TACA, e.g. with key authorization or some ssh-agent solution. If transfer 
+was done on a local system, the integrity of the transferred files is verified
+by comparing to the checksums calculated in the staging folder.
+
+Configuration
+-------------
+
+Configuration options affecting the delivery are specified under the 
+``deliver:`` section in the TACA configuration. Many configuration options
+can also be given on the command line. Option values given on the command line 
+takes precedence over option values specified in the configuration. 
+
+It is possible to use placeholders in paths in the configuration file. The
+placeholders will be evaluated and replaced with the corresponding value at 
+run-time, which is useful for specifying e.g. sample-specific paths. The syntax
+for a placeholder is the name in capital letters, surrounded by underscores, 
+e.g. ``_SAMPLEID_``. The placeholder will be replaced with the corresponding
+configuration option in lowercase letters, i.e. in the example above, this will 
+be ``sampleid``.
+
+Below is a description of common configuration options.
+
+``operator`` an email address to send notifications of errors occurring during
+delivery to
+
+``stagingpath`` path where files will be staged. *Required*
+
+``deliverypath`` path where the staged files will be delivered. If used together
+with option ``remote_host``, this will be the path on the remote system. 
+*Required*
+
+``files_to_deliver`` a list of tuples, where the first entry is a path 
+expression (this can be a file glob), pointing to a file or folder that should 
+be delivered, and the second entry is the path to where the matching file(s) or 
+folders (and contents) will be staged by symlinking. *Required*
+
+``hash_algorithm`` the algorithm that should be used for calculating the file
+checksums. Accepted values are algorithms available through the Python `hashlib`_ module.
+
+Below is a sample configuration snippet:
+
+.. code-block:: yaml
+
+    deliver:
+        deliverypath: /local/or/remote/path/to/transfer/destination
+        stagingpath: /path/to/stage
+        operator: notify-on-error@email.address
+        files_to_deliver:
+            -
+                - /expression/to/source/file/or/folder
+                - _STAGINGPATH_/the/file/or/folder/and/contents
+            -
+                - /expression/can/be/a/*/glob/as/well*
+                - _STAGINGPATH_/files/or/folders/matching/glob
+
+In addition, there is a dependency on the `ngi_pipeline`_ module, which requires
+its own configuration file. Please refer to the ngi_pipeline `documentation`_ 
+for details.
+
+Command line options
+--------------------
+
+The delivery script can be run for an entire project or for one or more samples
+in a project. The main delivery command is ``taca deliver`` and it takes further
+subcommands as described below. The deliver command accepts a number of options,
+described below.
```

Comment:
hmm this is good, the problem is that if you add/remove configuration options you'll have to remember to modify this template also. `taca deliver --help` should generate this, what about mentioning to execute this command instead of writing the options explicitly?